sdks/csharp.md

```diff
@@ -7,29 +7,73 @@ Coming Soon
 ### Initialize Bandwidth Client
 
 ```csharp
-var foo = new bar();
+using BandwidthSdk.Standard.BandwidthVoice;
+using BandwidthSdk.Standard.BandwidthVoice.Models;
+
+
+
+//create Configuration with credentials
+Configuration voiceConfig = new Configuration.Builder()
+			.WithBandwidthVoiceBasicAuthPassword("voice.password")
+			.WithBandwidthVoiceBasicAuthUserName("voice.username")
+			.WithEnvironment(Configuration.Environments.PRODUCTION)
+			.Build();
+
+//Activate the Client with the Configuration
+APIController voiceClient = new BandwidthVoiceClient(voiceConfig).Client;
+
 ```
 
 ### Create Phone Call
 
 ```csharp
-var foo = new bar();
+
+//Create the ApiCreateCallRequest object
+ApiCreateCallRequest callRequest = new ApiCreateCallRequest();
+
+callRequest.ApplicationId = "3-d-4-b-5";
+callRequest.To="+19999999999";
+callRequest.AnswerUrl= "https://test.com";
+callRequest.From="+17777777777";
+
+//Be aware that the Voice Client can throw exceptions 
+try {
+	voiceClient.CreateCall("account.id", callRequest);
+} catch (APIException e) {
+	WriteLine( e.Message );
+} catch (IOException e) {
+	WriteLine( e.Message );
+}
+
+
 ```
 
 ### Generate BXML
 
 ```csharp
-var foo = new bar();
+using BandwidthBXML;
```

Comment:
Is this the namespace in the SDK from APIMatic? I thought it was different. I could be wrong so ignore this if so